projects/kubernetes-sigs/cluster-api/build/create_manifests.sh

```diff
@@ -50,14 +50,26 @@ make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./config
 make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./bootstrap/kubeadm/config/default/manager_pull_policy.yaml"
 make set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./controlplane/kubeadm/config/default/manager_pull_policy.yaml"
 
+sed -i '' 's/^\( *\)args:$/&\
+\1- "--namespace=eksa-system"/' ./config/manager/manager.yaml
+
+sed -i '' 's/^\( *\)args:$/&\
+\1- "--namespace=eksa-system"/' ./bootstrap/kubeadm/config/manager/manager.yaml
+
+sed -i '' 's/^\( *\)args:$/&\
+\1- "--namespace=eksa-system"/' ./controlplane/kubeadm/config/manager/manager.yaml
+
 ## Build the manifests
 make release-manifests
 
 ## Build the development manifests
 make -C test/infrastructure/docker set-manifest-image \
 MANIFEST_IMG=$CAPI_REGISTRY_PREFIX/capd-manager MANIFEST_TAG=$IMAGE_TAG
 make -C test/infrastructure/docker set-manifest-pull-policy PULL_POLICY=IfNotPresent
-PATH="$(pwd)/hack/tools/bin:$PATH" make -C test/infrastructure/docker release-manifests 
+sed -i '' 's/^\( *\) - "--leader-elect"$/&\
```

Comment:
`sed -i ''`
This syntax of `sed` inplace will work only on MacOS. Using `gsed` instead would also us to use the same syntax as Nix systems.

```suggestion
SED=sed
if [ "$(uname -s)" = "Darwin" ]; then
    SED=gsed
fi
$SED -i '' 's/^\( *\) - "--leader-elect"$/&\
```